lib/src/board_color_scheme.dart

```diff
@@ -654,9 +654,19 @@ class BoardColorScheme {
       coordinates: true,
       orientation: Side.black,
     ),
-    lastMove: Color(0x809cc700),
-    selected: Color(0x6014551e),
+    lastMove: HighlightDetails(solidColor: Color(0x809cc700)),
+    selected: HighlightDetails(solidColor: Color(0x6014551e)),
     validMoves: Color(0x4014551e),
     validPremoves: Color(0x40203085),
   );
 }
+
+class HighlightDetails {
+  const HighlightDetails({
+    this.solidColor,
+    this.image,
+  });
```

Comment:
Here you could add an `assert` to ensure we don't forget to provide at least one of the arg.
Sth like:
```dart
 const HighlightDetails({
     this.solidColor,
     this.image,
   }) : assert(solidColor != null || image != null, 'You must provider either `solidColor` or `image`.');
```